### PR TITLE
Update CSS files.

### PR DESCRIPTION
**Pull request by XhmikosR**
* use shorthands when possible
* no need to use IDs with elements
* remove units from zero
* use lowercase color hex values consistently
* add the unprefixed `transition` property

### CODE REVIEW

File: site/nu-style.css [hunk 1]
```diff
@@ -860,7 +847,7 @@ dl.inputattrs .highlight {
 }
 
 #filters code {
-	background: #6F6;
+	background-color: #6f6;
```
Comment (xfq) [inline site/nu-style.css]: There are two `background-color` declarations in this rule.

File: site/nu-style.css [hunk 2]
```diff
@@ -70,38 +70,38 @@ textarea {
...
 }
 
-h1#title {
+#title {
 	font-family: "Myriad Web", "Myriad Pro", "Gill Sans", Helvetica, Arial, Sans-Serif;
-	background-color: #365D95;
+	background-color: #365d95;
 	color: #fdfdfd;
 	font-size: 1.6em;
 	padding: 0.430em;
```
Comment (xfq) [inline site/nu-style.css]: s/430/43/

File: site/style.css [hunk 3]
```diff
@@ -67,7 +67,7 @@ h3 {
 }
 
 h1 span {
-	color: #AAAAAA;
+	color: #aaaaaa;
 }
 
 fieldeset td, fieldeset th {
```
Comment (xfq) [inline site/style.css]: s/fieldeset/fieldset/

Comment (XhmikosR) [inline site/style.css]: No need to comment on unrelated stuff to my changes.

Comment (xfq) [inline site/style.css]: I apologize for my laziness. I should create a separate PR for this.

Comment (XhmikosR) [inline site/style.css]: Yeah, exactly my point :P

Nevertheless, I addressed your previous comments in this branch, but for any others just make a PR.
